File: 2MONTH_LESSON2-master/app/src/main/java/com/example/a2_month_lesson2/FirstFragment.java
package com.example.a2_month_lesson2;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

public class FirstFragment extends Fragment {

    final static String KEY_USERNAME = "username";
    private Button btnNext;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_first, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        initView();
        clickListener();
    }



    private void initView() {
        btnNext = requireActivity().findViewById(R.id.btn_next);
    }


    private void clickListener() {
        btnNext.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                nextFragment();
            }
        });
    }


    private void nextFragment() {
        Bundle bundle = new Bundle();
        bundle.putString(KEY_USERNAME, "alina003");
        SecondFragment fragment = new SecondFragment();
        fragment.setArguments(bundle);
        requireActivity().getSupportFragmentManager().beginTransaction()
                .addToBackStack(null).replace(R.id.container, fragment).commit();
    }
}

File: 2MONTH_LESSON5-master/app/src/main/java/com/example/a2_month_lesson5/FirstFragment.kt
package com.example.a2_month_lesson5

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.a2_month_lesson5.databinding.ActivityMainBinding
import com.example.a2_month_lesson5.databinding.FragmentFirstBinding

class FirstFragment : Fragment() {

    private lateinit var binding: FragmentFirstBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentFirstBinding.inflate(layoutInflater)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        binding.btnGo.setOnClickListener {
            requireActivity().supportFragmentManager.beginTransaction().replace(R.id.container, SecondFragment()).commit()
        }
    }
}